### ISSUE TITLE
Improvements to getting started experience for DS

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#1127]
Should we consider having some kind of a workflow that'll help users setup VSC for DS work.
E.g for if Jupyter doesn't start and this is the first time a user has attempted to run jupyter then:
* We recommend installing conda (if not installed)
* We check if conda has been setup correctly
* We suggest picking a jupyter interpreter from a conda environment...
* I.e. we walk the user through the process..

Today, we ask user to install jupyter if not found, then later we ask user to install ipykernel..
Basically we have prompts with solutions, but only when users run into an issue. 
I think we can improve this experience, rather than letting users stumble around and fix for themselves..

Kind of a Wizard approach, using standard prompts with steps like in Azure where they have `Step 1`, `Step 2`,, etc)

We could bring this prompt (that kicks off this workflow), when jupyter doesn't start or kernel fails, or the like.
